explainer.md

```diff
@@ -543,54 +543,55 @@ Beyond the core APIs described above, the WebXR Device API also exposes several
 
 ```js
 // Try to get a frame of reference where the floor is at Y = 0
-xrSession.requestFrameOfReference("stage").then((frameOfRef) => {
+xrSession.requestFrameOfReference("stage").then((coordSystem) => {
   // Will always succeed due to stage emulation. See the section titled
   // "Emulated stage frame of reference" for details.
-  xrFrameOfRef = frameOfRef;
+  xrCoordSystem = coordSystem;
 });
 
-// Use xrFrameOfRef as detailed above, but render the floor of the virtual space at Y = 0;
+// Use xrCoordSystem as detailed above, but render the floor of the virtual space at Y = 0;
 ```
 
-When using a stage `XRFrameOfReference` the device will frequently have a configured "safe area" that the user can move around in without fear of bumping into real world objects. WebXR can communicate the rough boundaries of this space via the `XRFrameOfReference.bounds` attribute. It provides a polygonal boundary given in the 'geometry' point array, which represents a loop of points at the edges of the safe space. The points are given in a clockwise order as viewed from above, looking towards the negative end of the Y axis. The shape it describes is not guaranteed to be convex. The values reported are relative to the stage origin, but do not necessarily contain it. The `bounds` attribute is null if the bounds are unavailable for the current frame of reference.
+When using a `stage` frame of reference the device will frequently have a configured "safe area" that the user can move around in without fear of bumping into real world objects. WebXR can communicate the rough boundaries of this space via the `XRSession`'s `getStageBoundsGeometry()` function. It provides a polygonal boundary given as an array of `XRStageBoundsPoint`s which represents a loop of points at the edges of the safe space. The points are given in a clockwise order as viewed from above, looking towards the negative end of the Y axis. The shape it describes is not guaranteed to be convex. The values reported are relative to the `XRCoordinateSystem` passed into the call, and do not necessarily contain the origin of that coordinate system. The bounds are unavailable for the device or given `XRCoordinateSystem`, an empty array will be returned.
```

Comment:
It seems fine to squash XRFrameOfReference itself into XRCoordinateSystem, as XRFrameOfReference doesn't do much over XRCoordinateSystem as currently defined, except to add some stage-specific attributes. (and stages are only one kind of frame)

However, it feels like a design smell that we already find ourselves stashing the stage-specific bounds data off on XRSession.  Especially as we start digging into anchors and other AR world understanding APIs, we are likely to have an increasing number of coordinate system flavors beyond stages that have extra metadata associated with them.

For example:
* Stages will have bounds
* Plane anchors will have extents, which will have change events
* Face anchors will have facial feature points that change each frame
* etc.

While stages will usually have just 1 instance, many of those other coordinate system flavors will have N instances, which will make it more awkward for us to stash such data elsewhere away from the coordinate system.  Rather than finding a one-off solution for stage attributes, we should audition an API pattern here that works for stages, anchors and beyond.

I dig in on ideas there a bit in my comment on [anchors issue 4](https://github.com/immersive-web/anchors/issues/4).